cpu/cortexm_common/periph/Makefile

```diff
@@ -1,3 +1,2 @@
 MODULE = cortexm_common_periph
```

Comment:
why are the common periph modules for the cortexm not compiled using the `periph.mk` makefile?

Reply:
fixed